src/IR.h

```diff
@@ -564,6 +564,12 @@ struct Call : public ExprNode<Call> {
         widening_shift_left,
         widening_shift_right,
         widening_sub,
+        declare_struct_type,
```

Comment:
Nit: the comment at the top of this enum asks to keep them sorted alphabetically.